packages/ilp-spsp-payout/src/index.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { PayoutConnection } from './lib/PayoutConnection'
+import { pluginFromEnvironment as makeILPPlugin } from 'ilp-plugin'
+
+const CLEANUP_TIMEOUT = 30 * 1000
+
+export class Payout {
+  private createPlugin: Function
```

Comment:
should be a function that takes no params and returns a Plugin rather than just generic function